internal/consumer/ackMgr.go

```diff
@@ -116,7 +117,7 @@ func (mgr *ackManager) Ack(id ackID) {
 func (mgr *ackManager) Nack(id ackID) {
 	mgr.Ack(id)
 	mgr.tally.Counter(metrics.KafkaPartitionNacks)
-	mgr.logger.Error("ackmgr received nack", zap.Int64("seq", id.msgSeq))
+	//mgr.logger.Error("ackmgr received nack", zap.Int64("seq", id.msgSeq))
```

Comment:
@venkat1109: why is this an logged as error?

Reply:
Safe to remove this log. Original intention was for validation / visibility.